Jenkinsfile

```diff
@@ -94,112 +93,117 @@ helpers.rootLinuxNode(env, {
         }
 
         stage("Test") {
-            def kbweb = null
             try {
-                // Install kbfsfuse first so we can start on dockerizing.
-                sh "go install github.com/keybase/kbfs/kbfsfuse"
-                sh "cp ${env.GOPATH}/bin/kbfsfuse ./kbfsfuse/kbfsfuse"
-                withCredentials([[$class: 'StringBinding', credentialsId: 'kbfs-docker-cert-b64', variable: 'KBFS_DOCKER_CERT_B64']]) {
-                    println "Building Docker"
-                    sh '''
-                        set +x
-                        docker build -t keybaseprivate/kbfsfuse --build-arg KEYBASE_TEST_ROOT_CERT_PEM_B64=\"$KBFS_DOCKER_CERT_B64\" kbfsfuse
-                    '''
+                if (startKbweb) {
+                    kbwebNodePrivateIP = httpRequest("http://169.254.169.254/latest/meta-data/local-ipv4").content
                 }
-                sh "docker save keybaseprivate/kbfsfuse | gzip > kbfsfuse.tar.gz"
-                archive("kbfsfuse.tar.gz")
 
+                // Trigger downstream builds
                 parallel (
-                    test_kbfs: {
-                        parallel (
-                            pull_mysql: {
-                                if (startKbweb) {
-                                    mysqlImage.pull()
-                                }
-                            },
-                            pull_gregor: {
-                                if (startKbweb) {
-                                    gregorImage.pull()
+                    //test_windows: {
```

Comment:
We still intend on moving them to Jenkins; just haven't ironed out the bugs yet.